summingbird-storm/src/main/scala/com/twitter/summingbird/storm/collector/TransformingOutputCollector.scala

```diff
@@ -0,0 +1,30 @@
+package com.twitter.summingbird.storm.collector
+
+import backtype.storm.spout.SpoutOutputCollector
+import java.util.{ List => JList }
+
+/**
+ * Created by pnaramsetti on 7/19/16.
+ *
+ * The KeyValueOutputCollector is used to unwrap the Value object when passed on by the Spout.
```

Comment:
you have called this `KeyValueOutputCollector` in the comment. Can we match the class?